### PR TITLE
Test

### PR DESCRIPTION
**Pull request by yuliana-r**
Added Bootstrap and Devise. 

Encountered error - not able to sign out. "Couldn't find user with 'id'=sign_out

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,69 +3,37 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
...
-
-  # Add speed badges [https://github.com/MiniProfiler/rack-mini-profiler]
-  # gem "rack-mini-profiler"
-
-  # Speed up commands on slow machines / big apps [https://github.com/rails/spring]
-  # gem "spring"
 end
 
 group :test do
-  # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
```
Comment (Moh-Yakoub) [inline Gemfile:68]: Did you remove those comments manually.

File: config/routes.rb [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
+  resources :users
+
   resources :mealplans
   resources :recipes
-  resources :users
   resources :ingredients
   resources :units
   resources :categories
 
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
```
Comment (Moh-Yakoub) [inline config/routes.rb:10]: Same here, is there a specific reason you removed those comments?

File: tmp/cache/assets/sprockets/v4.0.0/fo/fonnDj1dzRPf77ciLpeQV2S9J47BR1enuqQRYrzjSUQ.cache [hunk 3]
```diff
@@ -0,0 +1,3 @@
+[o:Set:
```
Comment (Moh-Yakoub) [inline tmp/cache/assets/sprockets/v4.0.0/fo/fonnDj1dzRPf77ciLpeQV2S9J47BR1enuqQRYrzjSUQ.cache:1]: Those files shouldn't be commented.

Probably you need to update your .gitignore file

**Comment by Moh-Yakoub** [comment]
In general looks good to me


left couple of comments.

General comments though:

1. Make sure you update your .gitignore file
2. Make sure you split the PRs to smaller units as it would be easier to review and update.
3. Make sure your PR description is very detailed ( same for the title ). Add test cases/screenshots in the description to help the reviewer

**Comment by Moh-Yakoub** [comment]
> Encountered error - not able to sign out. "Couldn't find user with 'id'=sign_out

Can you create an issue and explain how you exactly hit the issue, or at least attach couple of screenshots in the description

File: Gemfile (hunk 1 above)
Comment (yuliana-r) [reply Gemfile:68]: Yes, just added them back.

File: tmp/cache/assets/sprockets/v4.0.0/fo/fonnDj1dzRPf77ciLpeQV2S9J47BR1enuqQRYrzjSUQ.cache (hunk 3 above)
Comment (yuliana-r) [reply tmp/cache/assets/sprockets/v4.0.0/fo/fonnDj1dzRPf77ciLpeQV2S9J47BR1enuqQRYrzjSUQ.cache:1]: Just realised I was missing the .gitignore file. Added it now with the relevant template for Rails.

**Pull request merged**